### ISSUE TITLE
Bike routing: one way streets in wrong way

### ISSUE THREAD

**Issue by arnoulfos** [osmandapp/OsmAnd#18710]
### Describe the routing engine used (required)

- [ x] OsmAnd's in-app offline routing
- [ ] Any online routing provider (YOURS, openrouteservice, OSRM, etc.)
In bike routing in Paris area (France) osmand sends me systematically in one way streets in the wrong direction although in most cases (when there is no sign authorizing it for bikes) it is forbidden by the law and beside it may cause to get a fine from the police, it leads in some cases to a very dangerous situation when the street is tiny and there are many cars in the opposite directions.

### Describe your routing Profile (required)

Bike

### Describe your start and end points (required)

It occurs systematically in all routing across paris, for example start "Porte de pantin" destination "rue Georges Bizet"

### Describe the actual route (required)

I wrote it , it leads me to one way streets in the wrong way

### Describe the expected route (required)

Taking one way streets in the right way exwpt when authorized for bikes

### Describe what Maps you used (online or offline) (required)

- [x ] Offline maps offered within the OsmAnd app for download.  edition 10/1/2023
- [ ] Online (tile / raster) maps 

### Your Environment (required)

```
OsmAnd Version: 4.5.10 (12/9/2023)
Android/iOS version: 10
Device model: hisense a7
```

### Anything else? (optional)

Hope this is resolved shortly as it may lead to dangerous situations, this didn't occur in old versions of Osmand (5 years ago)

**Comment by letypequividelespoubelles** [comment]
See #17426 and https://forum.openstreetmap.fr/t/epidemie-de-dsc-a-anglet/15241

In France, roads with speed limit <30km/h are not one-way for bikes, unless specified.

**Comment by arnoulfos** [comment]
Can you give the reference of the law that allows it because I asked to policemen in Paris this week and they told me only when there is a sign authorizing the bike in the  opposite way

**Comment by arnoulfos** [comment]
Thanks a lot! This is a very recent law!  In case I would get a remark from the police in the future I know now which decree to refer this is very useful, thanks.

**Comment by arnoulfos** [comment]
@DmitryAlexei 
So at the light if the answers it appears that it is theorically correct concerning the law, but in the practice in a limited number of Parisian streets  it doesn t seem recommended especially at the hours of high traffic. I will report here some of those streets when I will have occasion to pass there again.

> @arnoulfos So what about that issue? Is it still actual? To reproduce it, we need the exact coordinates of the streets that are incorrectly taken into account for routing.

**Comment by DmitryAlexei** [comment]
Incorrect routing on one-way streets in France has been finally fixed in #19326
